kubeadm/flannel/flannel-host-gw.yml

```diff
@@ -74,6 +93,50 @@ data:
         ]
       }
     }
+  cni-conf-containerd.json: |
+    {
+      "cniVersion": "0.2.0",
+      "name": "cbr0",
+      "type": "flannel",
+      "capabilities": {
+        "portMappings": true,
+        "dns": true
+      },
+      "delegate": {
+        "type": "sdnbridge",
```

Comment:
@nagiesek
Should we be setting the forceBridgeGateway flag always since this is used by Flannel? I believe otherwise the .2 address which is reserved could be attempted to be assigned to pods?
```
        "optionalFlags" : {
            "forceBridgeGateway" : true
        }
```
Source: https://github.com/microsoft/windows-container-networking/blob/master/example/flannel_l2bridge.conf

I think similar may also be needed in the overlay CNI config?